encord/client.py

```diff
@@ -612,20 +613,31 @@ def add_users(self, user_emails: List[str], user_role: ProjectUserRole) -> List[
 
         return list(map(lambda user: ProjectUser.from_dict(user), users))
 
-    def copy_project(self, copy_datasets=False, copy_collaborators=False, copy_models=False) -> str:
+    def copy_project(
+        self,
+        copy_datasets=False,
+        copy_collaborators=False,
+        copy_models=False,
+        copy_labels=False,
+        copy_labels_options: Optional[CopyLabelsOptions] = None,
```

Comment:
Same comment here about the options and having `copy_labels` as `False`. Maybe we can have a type which makes it impossible to choose such an invalid state in first place.